tensorboard/plugins/pr_curve/summary_test.py

```diff
@@ -75,27 +75,29 @@ def test1Class(self):
 
     tensor_nd_array = tf.make_ndarray(tensor_event.tensor_proto)
 
-    # The tensor shape must be correct. The first dimension is the
-    # type of value (see documentation for the op). The 2nd dimension
-    # is the number of classes. The last dimension is the number of
-    # thresholds.
-    correct_shape = [6, 1, 10]
+    # The tensor shape must be correct. The first dimension is the value to
+    # compute (see documentation for the op or the comments below). The second
+    # dimension is the number of thresholds.
+    correct_shape = [6, 10]
```

Comment:
If you like, you can feel free to remove this whole check; `np.testing.assert_allclose` will fail if the shapes aren't compatible. But I'm not opposed to keeping it.